<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quest</class>
 <widget class="QDialog" name="quest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование тем</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/bg_image/images/Фон_2.PNG);</string>
  </property>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/bg_image/images/plus_PNG84.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="del">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/bg_image/Downloads/Remove-349235435.png);
image: url(:/bg_image/images/Remove-349235435.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>601</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bg_image/images/Фон_1.PNG);</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>599</width>
      <height>279</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLineEdit" name="inputpapka">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bg_image/images/White);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Введите название темы</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deletepapka">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bg_image/images/White);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Введите название темы</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Редактирование тем</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>350</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Редактирование названия теста</string>
   </property>
  </widget>
  <widget class="QPushButton" name="up1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
background-image: url(:/bg_image/images/White);</string>
   </property>
   <property name="text">
    <string>Увеличить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="down1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>450</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
background-image: url(:/bg_image/images/White);</string>
   </property>
   <property name="text">
    <string>Уменьшить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="up2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
background-image: url(:/bg_image/images/White);</string>
   </property>
   <property name="text">
    <string>Увеличить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="down2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>560</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
background-image: url(:/bg_image/images/White);</string>
   </property>
   <property name="text">
    <string>Уменьшить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="color1">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>450</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bg_image/images/White);
font: 14pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Цвет</string>
   </property>
  </widget>
  <widget class="QPushButton" name="color2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>560</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bg_image/images/White);
font: 14pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Цвет</string>
   </property>
  </widget>
  <widget class="QLabel" name="name1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>781</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:16pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Безопасность Жизнедеятельности</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="name2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>771</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Тест</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>450</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>560</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="red">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>771</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);
background-image: url(:/bg_image/images/White);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="apply">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>450</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);
background-image: url(:/bg_image/images/White);</string>
   </property>
   <property name="text">
    <string>Принять</string>
   </property>
  </widget>
  <widget class="QPushButton" name="apply_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>560</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);
background-image: url(:/bg_image/images/White);</string>
   </property>
   <property name="text">
    <string>Принять</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="red_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>771</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);
background-image: url(:/bg_image/images/White);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>add</zorder>
  <zorder>del</zorder>
  <zorder>scrollArea</zorder>
  <zorder>inputpapka</zorder>
  <zorder>label</zorder>
  <zorder>deletepapka</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>up1</zorder>
  <zorder>down1</zorder>
  <zorder>up2</zorder>
  <zorder>down2</zorder>
  <zorder>color1</zorder>
  <zorder>color2</zorder>
  <zorder>name1</zorder>
  <zorder>name2</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>red_2</zorder>
  <zorder>apply</zorder>
  <zorder>apply_2</zorder>
  <zorder>red</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
